compiler/rustc_ast/src/token.rs

```diff
@@ -710,7 +710,11 @@ impl Token {
         }
     }
 
-    pub fn glue(&self, joint: &Token) -> Option<Token> {
+    /// Checks if the current token is a multiple-character token (or glued like `==`, `<=`, `...`)
+    /// and returns the corresponding `TokenKind` if it is.
+    ///
+    /// if the current tokens is already complete, returns `None`.
+    pub fn check_is_multiple_char_token(&self, joint: &Token) -> Option<Token> {
```

Comment:
I prefer the name `glue`. Adding a method is fine, but the new name doesn't really explain that we're trying to combine `self` and `joint` into a new "glued"/merged/combined/joined token.

Reply:
Certainly it doesn't seem very practical. Thank you for your feedback.